xarray_sentinel/sentinel1.py

```diff
@@ -149,9 +193,10 @@ def find_avalable_groups(
                 )
             )
 
-        subgroups = list(METADATA_OPENERS.keys()) + burst_ids
+        subgroups = list(METADATA_OPENERS.keys()) + ["calibration"] + burst_ids
```

Comment:
`calibration` should be a `METADATA_OPENER`, although it has a slightly different interface. I will open a new issue for this.